### DISCUSSION TITLE
求助 安装失败

### DISCUSSION THREAD

**Discussion by shallwet** [lsky-org/lsky-pro#711] [问答]
Dropping all tables ............................................... 5ms FAIL
Installation error!

SQLSTATE[HY000] [2002] Connection refused (SQL: SHOW FULL TABLES WHERE table_type = 'BASE TABLE')
不知道是什么原因到安装这一步就报错了，数据库看了没有什么问题，这个connection refused也没有出现在mysql的error.log 里面

**Comment by 0xxb** [comment]
先创建你的数据库在安装

**Reply by shallwet** [reply]
已经创建数据库了，试了两次不同的数据库，然后用户也试了几次，root用户和自己创建的用户都试过都是这个报错

**Reply by 0xxb** [reply]
检查一下数据库连接地址

**Comment by YU-7** [comment]
我也是一样，补充一下docker的日志
![image](https://github.com/lsky-org/lsky-pro/assets/58320198/bf2ae677-5dab-49cc-a78b-cdca9859d22c)

**Reply by xiyu100** [reply]
> 我也是一样，补充一下docker的日志 ![image](https://private-user-images.githubusercontent.com/58320198/325557384-bf2ae677-5dab-49cc-a78b-cdca9859d22c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzODYzNDQsIm5iZiI6MTczMDM4NjA0NCwicGF0aCI6Ii81ODMyMDE5OC8zMjU1NTczODQtYmYyYWU2NzctNWRhYi00OWNjLWE3OGItY2RjYTk4NTlkMjJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMxVDE0NDcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhMzliYmM1YTNkMTZlOTc3NzU5ODViOGJhZmQ5ZjNlYmUzYTgxMjQ4OGIwYjg3NWQ1YjEyMWU0OGJmNGYwMjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.J8eA4qvpTMF1HddT-Y5_b1nkILBup-WOi2m0OLC0QK0)

解决了吗

**Reply by YU-7** [reply]
重新启动容器就好了
